src/Expander.hs

```diff
@@ -531,12 +533,12 @@ initializeKernel = do
               Just info
 
 
-            forkEstablishConstructors pdest d ctors
+            forkEstablishConstructors sc vp d ctors
```

Comment:
I now have to forward the `Scope` to `forkEstablishConstructors` because that's where the `DeclValidityPtr` gets filled.